src/parser/jobs/drg/modules/Weaving.ts

```diff
@@ -2,21 +2,30 @@ import {ActionKey} from 'data/ACTIONS'
 import {Weaving as CoreWeaving, Weave} from 'parser/core/modules/Weaving'
 import DISPLAY_ORDER from './DISPLAY_ORDER'
 
-const JUMPS: ActionKey[] = [
+// With the reduced animation lock, it's just stardiver that's the bad weave
+const JUMPS_ALL: ActionKey[] = [
+	'STARDIVER',
+]
+
+const JUMPS_600: ActionKey[] = [
 	'JUMP',
 	'HIGH_JUMP',
 	'SPINESHATTER_DIVE',
 	'DRAGONFIRE_DIVE',
-	'STARDIVER',
 ]
 
 export default class Weaving extends CoreWeaving {
 	static override displayOrder = DISPLAY_ORDER.WEAVING
 
-	private jumpIds = JUMPS.map(key => this.data.actions[key].id)
+	private jumpIds = JUMPS_ALL.map(key => this.data.actions[key].id)
+
+	// pre-6.1, all jumps are bad weaves
+	private jump600Ids = [...this.jumpIds, ...JUMPS_600.map(key => this.data.actions[key].id)]
```

Comment:
Alternatively, set the var and conditionally append to it, then just use it below as before. Something like this:

```ts
private jumps = JUMPS.map(key => this.data.actions[key].id)

if (this.parser.patch.before('6.1') {
	this.jumps.concat(JUMPS_600.map(key => this.data.actions[key].id))
}

Reply:
condensed it a bit, let me know if i've misinterpreted this note